src/map/enmity_container.cpp

```diff
@@ -171,6 +174,7 @@ void CEnmityContainer::UpdateEnmity(CBattleEntity* PEntity, int32 CE, int32 VE,
         VE = std::clamp((int32)(VE * bonus), 0, EnmityCap);
 
         m_EnmityList.emplace(PEntity->id, EnmityObject_t {PEntity, CE, VE, true, maxTH});
+		m_EnmityList.emplace(PEntity->id, EnmityObject_t {PEntity, CE, VE, true, maxCR});																		   
```

Comment:
don't add this line, modify the one above so it is

`m_EnmityList.emplace(PEntity->id, EnmityObject_t {PEntity, CE, VE, true, maxTH, maxCR});`